pkg/oc/generate/cmd/newapp_test.go

```diff
@@ -640,152 +640,3 @@ func TestBuildOutputCycleWithCircularTag(t *testing.T) {
-			allowNonNumericPorts: false,
-		},
-		{
-			strategy:             generate.StrategyDocker,
-			allowNonNumericPorts: false,
-		},
-	}
-
-	for _, test := range tests {
-		config := &AppConfig{}
-		config.Strategy = test.strategy
-		config.AllowNonNumericExposedPorts = test.allowNonNumericPorts
-
-		repo, err := app.NewSourceRepositoryForDockerfile("FROM centos\nEXPOSE 8080 NON_NUMERIC_PORT")
-		if err != nil {
-			t.Fatalf("Unexpected error during setup: %v", err)
-		}
-		repos := app.SourceRepositories{repo}
-
-		err = optionallyValidateExposedPorts(config, repos)
-		if err == nil {
-			t.Error("Expected error wasn't returned")
-
-		} else if !strings.Contains(err.Error(), "port number must be in range 0 - 65535") {
-			t.Errorf("Unexpected error: %v", err)
-		}
-	}
-}
-
-func TestExposedPortsAreValid(t *testing.T) {
-	tests := []struct {
-		dockerfile    string
-		expectedError string
-	}{
-		{
-			dockerfile:    "FROM centos\nEXPOSE 8080\nEXPOSE 8443",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE 8080/tcp\nEXPOSE 8443/udp",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nARG PORT=8080\nEXPOSE $PORT",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nENV PORT 8080\nEXPOSE $PORT",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nENV PORT=8080\nEXPOSE $PORT",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nENV PORT2=8080\nENV PORT1=$PORT2\nENV PORT=$PORT1\nEXPOSE $PORT",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nENV PORT2=8080 PORT1=$PORT2 PORT=$PORT1\nEXPOSE $PORT",
-			expectedError: "",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE 808080",
-			expectedError: "port number must be in range 0 - 65535",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE -1",
-			expectedError: "port number must be in range 0 - 65535",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE 808080/tcp",
-			expectedError: "port number must be in range 0 - 65535",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE -1/udp",
-			expectedError: "port number must be in range 0 - 65535",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE PORT",
-			expectedError: "port number must be in range 0 - 65535",
-		},
-		{
-			dockerfile:    "FROM centos\nEXPOSE 8080/xyz",
-			expectedError: "protocol must be tcp or udp",
-		},
-	}
-
-	for _, test := range tests {
-		config := &AppConfig{}
-		config.Strategy = generate.StrategyDocker
-		config.AllowNonNumericExposedPorts = false
-
-		repo, err := app.NewSourceRepositoryForDockerfile(test.dockerfile)
-		if err != nil {
-			t.Fatalf("Unexpected error during setup: %v", err)
-		}
-		repos := app.SourceRepositories{repo}
-
-		err = optionallyValidateExposedPorts(config, repos)
-		if err == nil && test.expectedError == "" {
-			continue
-		} else if !strings.Contains(err.Error(), test.expectedError) {
-			t.Errorf("Unexpected error: %v", err)
-		}
-	}
-}
```

Comment:
with removal of `AllowNonNumericExposedPorts`, these tests were redundant to the tests in `pkg/util/docker/dockerfile/dockerfile_test.go`